packages/ivy/ivy.1.3.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "gautier.hattenberger@enac.fr"
+version: "1.3.1"
+homepage: "http://www.eei.cena.fr/products/ivy"
+build: [
+  [make]
```

Comment:
There is no prefix in the Makefile for ivy-ocaml... it uses ocamlfind to install the libs...